src/main/java/solrocr/OcrHighlighter.java

```diff
@@ -439,11 +439,7 @@ public OcrHighlightResult[] highlightOcrFields(
           } catch (RuntimeException e) {
             // This catch-all prevents OCR highlighting from failing the complete query,
             // instead users get an error message in their Solr log.
-            String nSourcePtr = "n.a.";
-            if (content.getPointer() != null && content.getPointer().sources != null) {
-              nSourcePtr = String.valueOf(content.getPointer().sources.size());
-            }
-            log.error("Could not highlight ocr in '{}' sourceFiles", nSourcePtr, e);
+            log.error("Could not highlight ocr in source '{}'", content.getPointer(), e);
```

Comment:
This does not handle the case where the user didn't use source pointers, but stores the content in the index. Refer to https://github.com/dbmdz/solr-ocrhighlighting/pull/279#issuecomment-1243352194 on what this could look like.